src/Utils/SchemaExtender.php

```diff
@@ -513,7 +513,14 @@ public static function extend(Schema $schema, DocumentNode $documentAST, ?array
                 $schemaExtensions[] = $def;
             } elseif ($def instanceof TypeDefinitionNode) {
                 $typeName = isset($def->name) ? $def->name->value : null;
-                if ($schema->getType($typeName)) {
+
+                try {
+                    $type = $schema->getType($typeName);
+                } catch (Error $error) {
+                    $type = null;
+                }
+
+                if ($type) {
```

Comment:
if type what? I'd prefer to see more explicit and therefore readable condition. `$type !== null` ? 🤔